.travis.yml

```diff
@@ -31,4 +31,4 @@ script:
   - cat sql/updates/hotfixes/*.sql | mysql -utrinity -ptrinity hotfixes
   - mysql -uroot < sql/create/drop_mysql.sql
   - cd bin
-  - make -j 10
+  - make -j 8
```

Comment:
> Travis CI VMs run on 1.5 virtual cores. This is not exactly a concurrency, which allows to parallelize a lot, but it can give a nice speedup depending on your use case.
Which means 8-10 is too high anyway.